bigquery/dataset_test.go

```diff
@@ -325,7 +327,16 @@ func TestDatasetToBQ(t *testing.T) {
 			},
 			Location: "EU",
 			Labels:   map[string]string{"x": "y"},
-			Access:   []*AccessEntry{{Role: OwnerRole, Entity: "example.com", EntityType: DomainEntity}},
+			Access: []*AccessEntry{
+				{Role: OwnerRole, Entity: "example.com", EntityType: DomainEntity},
+				{Role: ReaderRole,
```

Comment:
I see in https://cloud.google.com/bigquery/docs/reference/rest/v2/datasets#Dataset.FIELDS.inlinedField_8 that "The role field is not required when this field is set. "

Do we want a test that verifies role can be omitted from Go?

Reply:
Work adjusting the unit test expectations, good spot.